Azure-Samples/ms-identity-javascript-react-tutorial
# demonstrate graph sdk usage

## Purpose

* This PR implements MS Graph JS SDK client with MSAL as custom authentication provider. This is demonstrated in chapter 2.1 call-graph sample.

## Does this introduce a breaking change?

```
[ ] Yes
[ x ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[ ] Bugfix
[ x ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone https://github.com/Azure-Samples/ms-identity-javascript-react-tutorial.git
cd ms-identity-javascript-react-tutorial
git checkout add-graph-sdk
cd Authorization-I/1-call-graph/SPA
npm install
```

* Test the code

Check [graph.js](https://github.com/Azure-Samples/ms-identity-javascript-react-tutorial/blob/e8aa8eecfd8471301d66d635a7b900fcb79c64e4/2-Authorization-I/1-call-graph/SPA/src/graph.js)

cc @tnorling
